### DISCUSSION TITLE
Why does this map configuration work with Map, but not with ProjectTo

### DISCUSSION THREAD

**Discussion by nivle** [LuckyPennySoftware/AutoMapper#3944] [NA]
```
CreateMap<Src, Dest>()
.ForMember(dest => dest.ChildOfDest, opt => opt.MapFrom((src, dest, destMember, context) => context.Mapper.Map<IList<ChildOfSrc>, IList<ChildOfDest>>(staticFunctionThatReturnsNewChildOfSrc(src.ChildOfSrc))));
```
```
await mapper.ProjectTo<Dest>(srcContext...).ToListAsync();
```
When I use ProjectTo I get:
```
Unable to create a map expression from . (Src) to IList`1.ChildOfDest(System.Collections.Generic.IList`1[ChildOfDest])

Mapping types:
Src -> Dest
Src -> Dest

Type Map configuration:
Src-> Dest
Src-> Dest

Destination Member:
ChildOfDest
```
Meanwhile this works fine with the above configuration:
```
mapper.Map<IList<Src>, IList<Dest>>(await srcContext.Include(s => s.ChildOfSrc).ToListAsync());
```

Is it because the items when using Mapper.Map are already loaded in memory?
Or should that code als be able to work with ProjectTo, and this is a bug?
Or is it because of some other reason?

I also tried using custom value resolvers while using ProjectTo and the result was the same.

**Comment by lbargaoanu** [comment]
https://docs.automapper.org/en/latest/Queryable-Extensions.html#supported-mapping-options

**Reply by nivle** [reply]
Thanks, that not supported list explains alot :D
